src/base/bittorrent/peerinfo.cpp

```diff
@@ -277,111 +277,126 @@ void PeerInfo::determineFlags()
         // d = Your client wants to download, but peer doesn't want to send (interested and choked)
         if (isRemoteChocked()) {
             m_flags += "d ";
+            m_flagsDescription += "d = ";
             m_flagsDescription += tr("interested(local) and choked(peer)");
-            m_flagsDescription += ", ";
+            m_flagsDescription += '\n';
```

Comment:
I would write:
```c++
m_flagsDescription += "d = "
    + tr("...") + '\n';
// or if the above doesn't work:
m_flagsDescription += QLatin1String("d = ")
    + tr("...") + '\n';
```
this also applies to other similar instances.

Reply:
Also appending newlines everywhere and chopping off in the end feels dumb for me, would be better to use a temporary `QStringList` to hold the strings and `join()` them in the end.